Description: Fixes CMAKE_CXX_FLAGS overwrite in CMakeLists.txt
 Normally CMake allows to specify custom compilation flags thru
 CMAKE_<LANG>_FLAGS. However, CMAKE_CXX_FLAGS is overwritten in Kadu's
 CMakeLists.txt, making it impossible to set those during cmake call.
Author: Patryk Cisek <patryk@debian.org>
---
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 71b4942..d8998b7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,7 +40,7 @@ else ()
 	set (COMMON_COMPILE_FLAGS "-Wall -Wextra -Wundef -Wunused -Wuninitialized -Wcast-align -Wpointer-arith -fno-common")
 	set (C_FLAGS "-Wwrite-strings")
 	set (CXX_FLAGS "-Woverloaded-virtual -Wnon-virtual-dtor -std=c++0x")
-	set (CMAKE_CXX_FLAGS "-std=c++0x")
+	set (CMAKE_CXX_FLAGS "-std=c++0x ${CMAKE_CXX_FLAGS}")
 
 	if (NOT WIN32)
 		set (COMMON_COMPILE_FLAGS "${COMMON_COMPILE_FLAGS} -fvisibility=hidden")
